server.js

```diff
@@ -1,13 +1,7 @@
 import express from "express";
 import cors from "cors";
 
```

Comment:
If you would like the fancy first page you could install `npm i express-list-endpoints` and then import it here `import getEndpoints from "express-list-endpoints`

And then on line 18